tutorials/capabilities/capabilities.md

```diff
@@ -87,45 +106,75 @@ by seL4's initialisation protocol, which consists of one CNode.
 
 ### CSpace addressing
 
-In order to refer to a capability, to perform operations on it, you must address the capability.
-There are two ways to address capabilities in the seL4 API. First is by invocation, the second
-is by direct addressing. Invocation is what we used to manipulate the registers of the root task's TCB, which
-we now explain in further detail.
+To refer to a capability and perform operations on it, you must *address* the
+capability. There are two ways to address capabilities in the seL4 API. First is
+by invocation, the second is by direct addressing. Invocation is a shorthand and
+it is what we used to manipulate the registers of the root task's TCB, which we
+now explain in further detail.
 
 #### Invocation
 
-On boot, the root task has a CNode capability installed as its *CSpace root*.
-An *invocation* is when a CSlot is addressed by implicitly invoking a thread's installed CSpace root.
-In the code example above, we use an invocation on the `seL4_CapInitThreadTCB` CSlot to read and write
-to the registers of the TCB represented by the capability in that specific CSlot.
+Each thread has a special CNode capability installed in its TCB as its *CSpace
```

Comment:
nit: because of the reformatting it's hard as a reviewer to focus on what has changed